packages/story-editor/src/components/library/panes/pageTemplates/pageTemplatesPane.js

```diff
@@ -157,9 +157,9 @@ function PageTemplatesPane(props) {
     setNextTemplatesToFetch,
   ]);
 
-  const handleToggle = () => {
-    setShowDefaultTemplates(!showDefaultTemplates);
-    localStore.setItemByKey(LOCAL_STORAGE_KEY, !showDefaultTemplates);
+  const handleSelect = (event, menuItem) => {
```

Comment:
looks like `event` isn't getting used. When targeting the second value passed through we typically do `(_, menuItem)`